Sources/Foundation/Collection/Collection+SafeIndex.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
```

Comment:
Please, not create new file. Move into `Collections+` file with `// MARK: - Safe`

Reply:
I've updated approach at all